solutions/USACO feb20-breedflip.mdx

```diff
@@ -15,33 +60,9 @@ sys.stdout = open("breedflip.out","w")
 N,a,b = input(),input(),input()
 s = [False]+[x != y for x,y in zip(a,b)] # difference list
 print(sum(1 if not x and y else 0 for x,y in zip(s,s[1:]))) # count occurrences of [False,True]
-```
-
-<!-- 
-```python
-fin, fout = open('breedflip.in'), open('breedflip.out', 'w')
-
-n = int(fin.readline())
 
-a = fin.readline()
-b = fin.readline()
-
-s = []
-for x in range(n):
-    if a[x] != b[x]:
-        s.append(1)
-    else:
-        s.append(0)
-
-def count(x,s):
-    count = 0
-    for f in range(len(s) - 1):
-        if s[f] == int(x[0]) and s[f+ 1] == int(x[1]):
-            count += 1
+```
 
-    return count
+<PySection/>
 
-s = [0] + s + [0]
-fout.write(str((count('01', s) + count('10', s))//2))
-```
- -->
\ No newline at end of file
+<LanguageSection/>
```

Comment:
These tags are wrong. They should instead be like `</LanguageSection>`

Reply:
I initially tried that, but it seems as though there is something else that was wrong. I switched it just to be safe.